packages/playground/src/pages/sandbox-pull-request-conversation-page.tsx

```diff
@@ -228,12 +232,13 @@ export default function SandboxPullRequestConversationPage() {
         <SandboxLayout.Column>
           <SandboxLayout.Content className="pl-0 pr-0">
             <PullRequestSideBar
+              addReviewers={noop}
+              usersList={undefined}
               // repoMetadata={undefined}
-              pullRequestMetadata={undefined}
+              pullRequestMetadata={{ source_sha: pullReqMetadata?.source_sha as string }}
```

Comment:
Any reason to typecast? `sha` should actually just be string?

Reply:
removed